ghcide/.ghci

```diff
@@ -1,29 +0,0 @@
-:set -Wunused-binds -Wunused-imports -Worphans -Wunused-matches -Wincomplete-patterns
```

Comment:
This made me mad as it took me 15 minutes to realize the issue.

It looks outdated to me, thus removed it